CHANGELOG.md

```diff
@@ -10,14 +10,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+[v0.11.0][v0.11.0_log]
+--------------
+
 ### Added
-- OE SDK Release packages can now be built on non-SGX and non-FLC machines.
+- Open Enclave SDK release packages can now be built on non-SGX and non-FLC machines.
 - Support for arbitrarily large thread-local data for SGX machines.
+- Experimental support for OpenSSL inside enclaves has been added while building the SDK from source.
+     - Use BUILD_OPENSSL flag while compiling the SDK
+     - [OpenSSLSupport.md](docs/OpenSSLSupport.md) documents supported options and configuration needed to OpenSSL inside an enclave.
 
 ### Changed
 - Rename the custom claims buffer added by oe_get_evidence from "custom_claims" to "custom_claims_buffer". Likewise, replace the `OE_CLAIM_CUSTOM_CLAIMS` definition for this name with `OE_CLAIM_CUSTOM_CLAIMS_BUFFER`.
+- Building SDK from source		
+      - HAS_QUOTE_PROVIDER cmake option has been removed. This is a continuation of the work in previous release to allow the same build of OE SDK to run on both FLC and non-FLC machines."
+      - Intel SGX EnclaveCommonAPI pacakges are no longer needed to build the SDK.
 
```

Comment:
Should sample change be reflected in CHANGELOG as well?

- The attestation functions in local_attestation/remote_attestation/attested_tls/host_verify samples are changed. Now they use attestation plugin APIs, defined in `attestation/attester.h and attestation/verifier.h` to generate and verify evidence.

Reply:
Done.